### PR TITLE
 Additional notes on leveraging setup:deploy functionality

### PR DESCRIPTION
**Pull request by GawainLynch**
ping @zomars

### CODE REVIEW

File: docs/nut-command/setup/setup-deploy.md [hunk 1]
```diff
@@ -5,20 +5,30 @@ level: intermediate
 setup:deploy
 ============
 
-Nut's `setup:deploy` is a simple tool to deploy a build of the current site 
+<p class="note"><strong>Note:</strong> The command is provided by a separate
+package called <code>bolt/simple-deploy</code> that is installed by default
```
Comment (bobdenotter) [inline docs/nut-command/setup/setup-deploy.md]: "<code>bolt/simple-deploy</code>, that" needs a comma, right?

Comment (GawainLynch) [reply docs/nut-command/setup/setup-deploy.md]: Nope, _"that is installed by default"_ is a restrictive clause in this case.

Comment (bobdenotter) [reply docs/nut-command/setup/setup-deploy.md]: Alright, then! 👍

File: docs/nut-command/setup/setup-deploy.md [hunk 2]
```diff
@@ -5,20 +5,30 @@ level: intermediate
...
+project installs.</p>
+
+Nut's `setup:deploy` is a simple tool to deploy a build of the current site
 from a local workstation to a (S)FTP enabled destination host.
 
 To use this tool, on your development machine **only** you can create a file
 named `.deploy.yml`.
 
+<p class="note"><strong>Note:</strong> This command will not appear in the list
+of available commands if the <code>.deploy.yml</code> file is not present.</p>
```
Comment (zomars) [inline docs/nut-command/setup/setup-deploy.md]: That's what I was missing... Thanks for clarifying!

Comment (GawainLynch) [reply docs/nut-command/setup/setup-deploy.md]: Yeah, massive facepalm on my part

**Pull request merged**